FRW-7741: limitation guides for DMS (#2790)
spryker/spryker-docs: docs/pbc/all/dynamic-multistore/202311.0/base-shop/dynamic-multistore-feature-overview.md

The following image shows the ability to call API endpoints with the exemplary "SECOND" store selected in the API call itself.

![storefront-api](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/storefront-api.png)

> 

Reviewer: I have doubts regarding 202311.0 version. 
Apparently, this should be discussed with the PO to align the next release date and move the updated doc there or get the confirmation that we leave it as it is. 
But as I see we have a fresher version for the doc under 202404.0

Author: You are right. Moved to 202404.0

## Adding of a new store via data import

Example for DE store configuration:

**data/import/common/{REGION}/store.csv**

```csv
name
DE
AT
```

| Column     | REQUIRED | Data Type | Data Example | Data Explanation |
|------------| --- | --- | --- | --- |
|name        |mandatory |string | DE | Define store name. |

Configure a new store as described here: [Import data for store](/docs/pbc/all/dynamic-multistore/202311.0/base-shop/install-and-upgrade/install-features/install-dynamic-multistore.html#import-data)

> Adding of a new store via Back Office is easier and faster way but in this case you should add this store at all environments. 

Reviewer: ```suggestion
Adding a new store via the Back Office is easier and faster, but in this case, you should add this store across all environments. 
```

> Data import allows you to add and configure a new store once and deploy it to all environments. 

Reviewer: ```suggestion
Data import allows you to add and configure a new store once and deploy it across all environments. 
```

Note that when you add a new store you should also assign all store-related entities to it for enabling them for customers:
- Products
- Categories
- CMS entities
- Prices. See [Install Dynamic Multistore + the Prices feature](/docs/pbc/all/dynamic-multistore/202311.0/base-shop/install-and-upgrade/install-features/install-dynamic-multistore-prices-feature.html#install-feature-core)
- etc.

> 

Reviewer: Do we have anywhere described the **minimal** scope of all needed setups to make a new store alive for Backoffice and data import? I mean if I were a customer it would be beneficial to have a doc with all the steps that guarantee I follow it and have a live environment with products. For example, assign the category, assign the warehouse and that's all that you need without `etc` so I don't have to guess what is missed.
If yes I would add a link here otherwise to describe both for the Back Office and Data import. 

@asmarovydlo @gechetspr what do you think?

Author: Created a page with minimal set of data for import

To avoid manual assignment of entities to the store you can use the data import for store-related entities as well.

## What is changed when DMS is enabled?

There is a list of changes that are applied to the project when DMS is enabled:
- Urls for Yves, Back Office, Merchant Portal and Glue contain region instead of store name.

>     - Example for local environment: https://backoffice.eu.spryker.local instead of https://backoffice.de.spryker.local 

Reviewer: ```suggestion
    - Example for local environment: https://backoffice.eu.mysprykershop.com instead of https://backoffice.de.mysprykershop.com 
```

- RabbitMQ virtual hosts contain region instead of store.

>     - For example eu-docker instead of de-docker.

Reviewer: We could include some before-and-after pictures to show how it looked initially and how it appears now.

- Jenkins job names contain region instead of store. 

>     - For example EU_queue-worker-start instead of DE_queue-worker-start.

Reviewer: Same here

- Elasticsearch indexes contain store as a part of the index name for DMS enabled and disabled modes.

> - Redis keys contain store as a part of the key name for DMS enabled and disabled modes.

Reviewer: Same, please provide examples

- When DMS is enabled customer can switch between available stores for a region.

>     - When customer changes a store it's set to a query param `_store`.

Reviewer: 1) Let's mention and add a picture of the switcher on Yves to switch between stores.

2) Also probably we can say a little bit that the Back Office has become store-agnostic now. 

3) Should we include some key links at the end of the document, such as instructions for configuring DS with the Glue API?  cc: @asmarovydlo @gechetspr

Reviewer 2: Please add the links
